client/src/js/components/bhLatestInvoice.js

```diff
@@ -0,0 +1,52 @@
+angular.module('bhima.components')
+.component('bhLatestInvoice', {
+  controller   : LatestInvoice,
+  controllerAs : '$ctrl',
+  templateUrl  : 'partials/templates/bhLatestInvoice.tmpl.html',
+  bindings: {
+    enablePatientDetails : '<',  // bind boolean (true|false) : Enable patient details option
+    enableOptionBar      : '<',  // bind boolean (true|false) : Enable option for add, display list or display thumbnail in a bar
+    enableSearch         : '<',  // bind boolean (true|false) : Enable search bar option
+    display              : '@',  // bind (list|thumbnail)  : Display either in list or thumbnail mode
+    height               : '@',  // bind the height of list of contents
+    debtorUuid          : '<'  // Required patient uuid
+  }
+});
+
+LatestInvoice.$inject = [
+  'PatientService', 'NotifyService'
+];
+
+/**
+ * Find Document Component
+ * This component is responsible for displaying the Latest Invoice 
+ */
+function LatestInvoice(Patient, Notify) {
+  var vm = this;
+
+  /** global variables */
+  vm.session = {
```

Comment:
It doesn't look like you need any of these variable...  what does `enableSearch` do?  Please do not just copy code - the code must mean something.